RatS/filmtipset/filmtipset_ratings_parser.py

```diff
@@ -0,0 +1,82 @@
+#     <option value="betyg">Filmbetyg
+#     </option><option value="kommentarer">Filmkommentarer
+#     </option><option value="recensioner">Recensioner
+#     </option></select><br>
+#     <input type="submit" id="submit-export" name="submit-export" value="Exportera data" class="button"><br>
+# </form>
+
+class FilmtipsetRatingsParser(RatingsDownloader):
+    def __init__(self, args):
+        super(FilmtipsetRatingsParser, self).__init__(Filmtipset(args), args)
+        # TODO: due to timezone and potential racecondition, it's probably more safe to
+        #       find/search file in download folder
+        date = datetime.datetime.fromtimestamp(time.time()).strftime('%Y-%m-%d')
+        self.downloaded_file_name = 'ft_betyg_{}.csv'.format(date)
+        self.csv_delimiter = ';'
+
+    # TODO remove when filmtipset has cleaned up thier "csv" file
+    @staticmethod
+    def _repair_line(line):
+        output = re.search(r'([0-9]{4}-[0-9]{2}-[0-9]{2}),(.*)(;[\-0-9]*;[1-5])$', line)
+        if output is None:
+            return line
+        group2 = output.group(2).replace('"', '""')
+        newline = '{};"{}"{}\n'.format(output.group(1), group2, output.group(3))
+        return newline
+
+    @staticmethod
+    def _repair_csv_file(filepath):
+        copyfile(filepath, filepath + '.bak')
+        for line in fileinput.input(filepath, inplace=True): 
+            print(FilmtipsetRatingsParser._repair_line(line),end="")
+
+    def _parse_ratings(self):
+        self._download_ratings_csv()
+        self._rename_csv_file(self.downloaded_file_name)
+        filepath = os.path.join(self.exports_folder, self.csv_filename)
+        self._repair_csv_file(filepath)
+        self.movies = self._parse_movies_from_csv(filepath)
+
+    def _call_download_url(self):
+        self.site.browser.find_element_by_xpath('//input[@id="submit-export"]').click()
+        self.site.browser.get('https://www.filmtipset.se/export-data.php?q=')
+
+    #VoteDate;MovieTitle;IMDB;Score
+    def _convert_csv_row_to_movie(self, headers, row):
+        movie = dict()
+        movie['title'] = row[headers.index("MovieTitle")]
+        movie[self.site.site_name.lower()] = dict()
+        filmtipset_rating = int(row[headers.index("Score")])
+        movie[self.site.site_name.lower()]['my_rating'] = filmtipset_rating
+        self._extract_imdb_informations(movie, row[headers.index("IMDB")], filmtipset_rating)
+
+        return movie
+
+    @staticmethod
+    def _extract_imdb_informations(movie, imdb_id, filmtipset_rating):
+        try:
+            i = int(imdb_id)
+            if i < 1:
+                raise ValueError
+        except ValueError:
+            return
+
+        imdb = dict()
+        imdb['id'] = 'tt{imdb_id_number:07d}'.format(imdb_id_number=int(imdb_id))
+        imdb['url'] = 'https://www.imdb.com/title/{imdb_id}'.format(imdb_id=imdb['id'])
+        imdb['my_rating'] = int(filmtipset_rating * 2)
```

Comment:
I transform the rating to 1 - 10 here and it's not clear to me if this is the right approach. But it feels more correct than adding the 1 - 10 rating in the filmtipset dict.

Reply:
Semantically, the `movie['imdb']['my_rating']` is meant to be the rating I gave to the move on the IMDB page. 
For a transfer from Filmtipset to e.g. Trakt, this value would be ignored and the `movie['filmtipset']['my_rating']` is the source value for rating the same movie on the destination page.

```suggestion
```